integration_tests/test_users_real_api.py

```diff
@@ -245,6 +264,8 @@ def test_multi_user(self):
         # Clean up. Delete the users
```

Comment:
I'm not totally sure what this is running against in an integration test context, but as cleanup you might want to move these deletions to a fixture or a finally clause such that if anything fails above these will not leave behind the residue or the created users etc.

Reply:
Hmmm, maybe we can have a tear-down method that deletes [any user or group created during the integration test] iff any part of the integration test fails. I think I would have it use the Python API directly instead of calling our CLI. What does @zaro0508 think?

Reply:
Also, I think we would want to do that as part of another PR. I can open an issue as soon as we agree on what to do

Reply:
My thought on this is the same as what I said in my previous review https://github.com/Sage-Bionetworks/jccli/pull/70#discussion_r492323843  `.. Just document what a config file looks like and tell the user to create it themselves`

Reply:
Sorry, @zaro0508 , I'm not sure I understand how your comment applies to this proposed change. Are you saying we should not try to do too much work for our users (developers) and therefore we should not have a tear-down method that cleans up their JC account in case of failure?